src-java/flowmonitoring-topology/flowmonitoring-storm-topology/src/main/java/org/openkilda/wfm/topology/flowmonitoring/service/ActionService.java

```diff
@@ -179,6 +214,14 @@ public void sendFlowRerouteRequest(String flowId) {
                 log.info("Sending flow '{}' reroute request.", flowId);
                 carrier.sendFlowRerouteRequest(flowId);
             }
+        } else if (haSubFlowRepository.findById(flowId).isPresent()) {
+            HaFlow haFlow = haSubFlowRepository.findById(flowId).get().getHaFlow();
+            if (haFlow != null) {
+                if (LATENCY_BASED_STRATEGIES.contains(haFlow.getPathComputationStrategy()) && isReactionsEnabled()) {
+                    log.info("Sending HA-flow '{}' reroute request.", haFlow.getHaFlowId());
+                    carrier.sendFlowRerouteRequest(haFlow.getHaFlowId());
```

Comment:
I think it was a part of https://github.com/telstra/open-kilda/pull/5204. Looks very weird.

Reply:
The part of #5204 was to implement handling of HA-flow this request. You just need to send right request.
We have 3 types of requests:
1. FlowReroute
2. YFLowReroute
3. HaFlowReroute

we do not send Y flow reroute here, because Y flow is just a facade for 2 read flows. 
But HaFlow is separate entity. We can't reroute one of HA-subflows, because HA flow can't be divided into two sub flows.

Reply:
Yeah, you are right, I was blind, sorry. Will update by analogy with the regular flow. Thanks!